system76_backlight_manager/keyboard_backlight.py

```diff
@@ -16,6 +16,7 @@ class KeyboardBacklight:
         "darp6": SINGLE_BACKLIGHT_PATH,
         "oryp6": SINGLE_BACKLIGHT_PATH,
         "oryp4": FOUR_BACKLIGHT_PATH,
+        "pang10": SINGLE_BACKLIGHT_PATH,
```

Comment:
```suggestion
        "pang10": LEGACY_SINGLE_BACKLIGHT_PATH,
```

Reply:
Ah sorry! Thanks for the catch!

Reply:
Do we have commands that I can share with my coworker to test this project?

Reply:
This should do the trick to set up dev and run the software:
```
make install-dev
source .venv/bin/activate
python system76_backlight_manager
```

or you can install the software:

```
make install
sudo systemctl start system76-backlight-manager.service
```

Reply:
We got this error:

`no rule to make target 'install-dev'. Stop.`